src/Project/functions/createNodeModulesPathMapping.ts

```diff
@@ -5,20 +5,29 @@ import { realPathExistsSync } from "Shared/util/realPathExistsSync";
 export function createNodeModulesPathMapping(nodeModulesPath: string) {
 	const nodeModulesPathMapping = new Map<string, string>();
 	if (fs.pathExistsSync(nodeModulesPath)) {
-		// map module paths
-		for (const pkgName of fs.readdirSync(nodeModulesPath)) {
-			const pkgPath = path.join(nodeModulesPath, pkgName);
-			const pkgJsonPath = realPathExistsSync(path.join(pkgPath, "package.json"));
-			if (pkgJsonPath !== undefined) {
-				const pkgJson = fs.readJsonSync(pkgJsonPath) as {
-					main?: string;
-					typings?: string;
-					types?: string;
-				};
-				// both "types" and "typings" are valid
-				const typesPath = pkgJson.types ?? pkgJson.typings ?? "index.d.ts";
-				if (pkgJson.main) {
-					nodeModulesPathMapping.set(path.resolve(pkgPath, typesPath), path.resolve(pkgPath, pkgJson.main));
+		// go through each org
+		for (const scope of fs.readdirSync(nodeModulesPath, { withFileTypes: true })) {
```

Comment:
A bit concerned about this, there could be a _lot_ of folder in node_modules from things like eslint and typescript.

Could we grab candidates from rojo config or typeRoots maybe?